<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>checklist_creation</class>
 <widget class="QDialog" name="checklist_creation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>896</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Создание Чек листа</string>
  </property>
  <widget class="QPushButton" name="btn_save_list">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>850</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="main_name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Название :</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>70</y>
     <width>591</width>
     <height>761</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>60</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>180</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>120</number>
   </attribute>
   <column>
    <property name="text">
     <string>Название</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Описание</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Время</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Текущие задания</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Название:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="task_name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Описание:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="description">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>481</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>610</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Время выполнения:</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="task_time">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>640</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="displayFormat">
    <string>H:mm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Добавить задание в чек лист</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add_task">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>730</y>
     <width>481</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLabel" name="task_warning_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>680</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="warning_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>790</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>850</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delete_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>860</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255);
bold;</string>
   </property>
   <property name="text">
    <string>Удалить Лист</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
